chirp/drivers/baofeng_uv17.py

```diff
@@ -65,6 +64,11 @@ def _download(radio):
     radio.status_fn(status)
 
     for block_number in radio.BLOCK_ORDER:
+        if block_number not in memory_map:
```

Comment:
Ah, this is the answer to my question about changing the dec to hex. You're just using those as indexes into the array, so it doesn't matter what they are. I thought you were multiplying those values, like on L73 below. But, I see you're just *indexing* and using that, so makes sense.

Reply:
They are in BCD format. I was converting them, but it's pointless since I am only using the values for indexing, indeed.